model/job-management/src/main/proto/org/apache/beam/model/job_management/v1/beam_expansion_api.proto

```diff
@@ -53,6 +54,10 @@ message ExpansionRequest {
   // coders for the output PCollections. Note that the request
   // may not be fulfilled.
   map<string, string> output_coder_requests = 4;
+
+  // (Optional) A set of Pipeline Options that should be used
+  // when expanding this transform.
+  google.protobuf.Struct pipeline_options = 5;
```

Comment:
@chamikaramj just checking this with you

Reply:
+1. We've been thinking about propagating other PipelineOptions to expansion as well (which we can piggy-back on top of this feature, on a case-by-case basis).